modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
I notice that the SSL, proxy and timeout methods have been removed. Can you share more with me how a developer will set SSL, proxy, timeout in the new client?